build.gradle.kts

```diff
@@ -61,5 +64,30 @@ subprojects {
         withType<io.gitlab.arturbosch.detekt.Detekt> {
             jvmTarget = JavaVersion.VERSION_1_8.toString()
         }
+
+        val dokka by getting(org.jetbrains.dokka.gradle.DokkaTask::class) {
+            outputFormat = "html"
+            outputDirectory = javadoc.get().destinationDir!!.absolutePath
```

Comment:
You may want to add some other stuff here. If I recall, Dokka default to JDK 6 for linking purposes.

Do you document anything that references a library, like Kotlin Poet? You would have to specify a link to those docs here.

Reply:
Added `jdkVersion = 8` in bae22db. Right now no external libraries are referenced, since the processor is entirely internal.